ansible/group_vars/gelato-ha.yml

```diff
@@ -22,6 +22,9 @@ dummy:
 # GENERAL #
 ###########
 
+# User permitted to access the Kubernetes Cluster to deploy the multicloud services.
+k8s_user: K8S_USER 
+
 # These variables should not be modified manually. 
 # These are populated by installer in a Multi-Cloud HA installation. 
 gelato_ha_api_ip: MULTICLOUD_API_IP
```

Comment:
I think we can keep this default to loopback IP 127.0.0.1 and the port to the default multicloud ports.

Reply:
ok, will do.